src/popup/components/IconBoxed.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="icon-boxed">
+  <div
+    class="icon-boxed"
+    :class="[variant]"
```

Comment:
To make it easier to debug I suggest to:
```suggestion
    :class="[`variant-${variant}`]"
```